bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManagerImpl.java

```diff
@@ -599,13 +607,20 @@ private void initializeHandler(Thing thing) {
             }
             ThingType thingType = getThingType(thing);
             applyDefaultConfiguration(thing, thingType);
-            if (isInitializable(thing, thingType)) {
-                setThingStatus(thing, buildStatusInfo(ThingStatus.INITIALIZING, ThingStatusDetail.NONE));
-                doInitializeHandler(thing.getHandler());
+
+            if (isPersistedAsEnabled(thing.getUID())) {
```

Comment:
This should simply be an `isEnabled` call.

Reply:
The public method `isEnabled(ThingUID)` will be with the same signature, so will have to keep this one as it is, or we can think of some other name.